lib/flag-filter.js

```diff
@@ -184,11 +184,15 @@ FlagFilter.prototype._transform = function (vm, enc, cb) {
 		vm.ptrname = vm.tags[DOCKER_PREFIX + INST_PTR_TAG];
 	}
 
-	/* And enable VMs owned by admin that have a service attached. */
-	if (vm.owner.login === 'admin' && vm.services.length > 0) {
-		vm.listInstance = true;
-		vm.listServices = true;
-		vm.reasons.push('admin');
+	/*
+	 * And enable VMs owned by admin or poseidon that have a service
+	 * attached.
+	 */
+	if ((vm.owner.login === 'admin' || vm.owner.login === 'poseidon')
+	    && vm.services.length > 0) {
```

Comment:
Ok, one last nit to pass the style checker. This needs to be formatted like this:

```
        if ((vm.owner.login === 'admin' || vm.owner.login === 'poseidon') &&    
            vm.services.length > 0) {                                           
                vm.listInstance = true;                                         
                vm.listServices = true;                                         
                vm.reasons.push(vm.owner.login);                                
        }                                                                       
```

* Line 191 indented with 2 tabs
* The `&&` continuation needs to be at the end of line 191, not the beginning of line 192
* Line 192 indented with 2 tabs and 4 spaces
* Lines 193-195 indented with 3 tabs